geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -267,11 +268,47 @@ class DateFunctionFactory extends TransformerFunctionFactory {
 }
 
 class GeometryFunctionFactory extends TransformerFunctionFactory {
-  override def functions = Seq(pointParserFn)
+  override def functions = Seq(pointParserFn, lineStringParserFn, polygonParserFn, geometryParserFn)
 
   val gf = JTSFactoryFinder.getGeometryFactory
   val pointParserFn = TransformerFn("point") { args =>
-    gf.createPoint(new Coordinate(args(0).asInstanceOf[Double], args(1).asInstanceOf[Double]))
+    args.length match {
+      case 1 =>
+        args(0) match {
+          case g: Geometry => g.asInstanceOf[Point]
```

Comment:
So the geo-json automatically gets converted to a geometry by the time it hits this code?